### PR TITLE
Run command

### LINKED ISSUE #94: Implement the runtime run command
`run` creates and starts a container

### PR DESCRIPTION
**Pull request by devimc**
Run command implementation

fixes #94

### CODE REVIEW

File: run.go [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+	},
+}
+
+func run(context *cli.Context) error {
+	runtimeConfig, ok := context.App.Metadata["runtimeConfig"].(oci.RuntimeConfig)
+	if !ok {
+		return errors.New("invalid runtime config")
+	}
+
+	err := Create(context, runtimeConfig)
```
Comment (sboeuf) [inline run.go]: Use
```go
if err := create(); err != nil {
```
instead of
```go
err := create()
if err != nil {
```

File: run.go [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+	if err != nil {
+		return err
+	}
+
+	pod, err := Start(context.Args().First())
```
Comment (sboeuf) [inline run.go]: s/Start/start/

File: run.go [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+	detach := context.Bool("detach")
+	if detach == false {
+		pid := pod.GetContainers()[0].GetPid()
+		p, err := os.FindProcess(pid)
+		if err != nil {
+			return err
+		}
+
+		_, err = p.Wait()
```
Comment (sboeuf) [inline run.go]: I would retrieve the ProcessState object and print it through the error returned with something like this:
```go
return fmt.Errorf("Process state %s: %s", processState.String(), err)
```

**Comment by devimc** [comment]
@sboeuf fixed

**Comment by sboeuf** [comment]
@devimc cool thanks. You should factorize the flags in common between `run()` and `create()` but this can be done in a follow up PR !
LGTM

File: run.go [hunk 4]
```diff
@@ -0,0 +1,98 @@
+// Copyright (c) 2014,2015,2016 Docker, Inc.
```
Comment (sameo) [inline run.go]: Any reason why we're adding a Docker copyright ?

Comment (sboeuf) [inline run.go]: Good catch, we should remove this, it is coming from the copy/paste from runc I guess.

File: run.go [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+		return err
+	}
+
+	pod, err := start(context.Args().First())
+	if err != nil {
+		return err
+	}
+
+	detach := context.Bool("detach")
+	if detach == false {
```
Comment (dlespiau) [inline run.go]: `if !detach`. I've seen those comparaisons of bool to true or false literals a bit all over the place and, while I don't have many pet peeves, this one may count as one.

Note that gometalinter will complain about it as well.

File: run.go (hunk 4 above)
Comment (devimc) [inline run.go]: removing

File: run.go (hunk 5 above)
Comment (devimc) [inline run.go]: yes, I saw this ```x == false``` in several files, I thought that was ok

Comment (sboeuf) [inline run.go]: Yep that's the way I was testing boolean variable. But I think we have to change that if gometalinter complains about it.

**Comment by jodh-intel** [comment]
lgtm

**Pull request merged**